crypto/representationproofs/representation.go

```diff
@@ -0,0 +1,148 @@
+package representationproofs
+
+import (
+	"github.com/xlab-si/emmy/crypto/common"
+	"github.com/xlab-si/emmy/crypto/groups"
+	"math/big"
+	"fmt"
+)
+
+func ProveKnowledgeOfRepresentation() bool {
+	group, err := groups.NewSchnorrGroup(256)
+	if err != nil {
+		fmt.Println(err)
+		fmt.Printf("Error when generating SchnorrGroup")
+		return false
+	}
+	var bases []*big.Int
```

Comment:
If we know in advance that a slice will have precisely 3 elements, why not just use a fixed-size array? 
Here's how L17-L22 would be if we used an array:

````go
var bases [3]*big.Int
for i := 0; i < len(bases); i++ {
 r := common.GetRandomInt(group.Q)
 base := group.Exp(group.G, r)
 bases[i] = base
}
````

The same goes for the `secrets` slice below.